RealtimeCSG/Assets/Plugins/RealtimeCSG/Editor/Scripts/Control/Managers/UpdateLoop.cs

```diff
@@ -64,9 +64,10 @@ void Initialize()
 			EditorApplication.hierarchyWindowChanged	+= OnHierarchyWindowChanged;
 #endif
 
-#if UNITY_2018_3_OR_NEWER
+#if UNITY_2018_3_OR_NEWER && !UNITY_2020_1_OR_NEWER
```

Comment:
what about < 2018.3 ?

Reply:
this is how it was written already, so im not sure what you mean?

Reply:
i realized what you had meant with this, and it has been fixed with the latest commit